edenia/eden-spend-explorer
# feat(webapp): expenses across election rounds

### Expenses across election rounds, all elections, and delegates

### What does this PR do?

- Resolve #165
- Resolve #25 
- Resolve #26 
- Resolve #28 
- Resolve #148 
- Resolve #27

### Steps to test

1. Run the project and open localhost:3000 in your browser
2. click on income and expense and see the new features

#### CheckList
